Android/app/src/main/java/io/github/project_travel_mate/MainActivity.java

```diff
@@ -171,6 +171,7 @@ protected void onCreate(Bundle savedInstanceState) {
                     break;
             }
         }
+
```

Comment:
revert this change too

Reply:
ok, done

Reply:
not yet reverted. This file is still here